support-models/src/main/scala/com/gu/support/workers/Products.scala

```diff
@@ -57,25 +57,14 @@ case class GuardianWeekly(
 
 object ProductType {
   import com.gu.support.encoding.CustomCodecs._
-  implicit val decoderDigital: Decoder[DigitalPack] = deriveDecoder[DigitalPack]
-    .prepare(_.withFocus(_.mapObject(_.checkKeyExists("readerType", Json.fromString("Direct")))))
```

Comment:
we used to accept a missing readerType, but it looks like the frontend does send it.  So I have dropped that functionality as the standard `DiscriminatedType` doesn't have facility for special cases at the moment.
It wouldn't be too hard to add the extra complexity, but I don't know if it's too big of a risk to drop that?  Thoughts please.

Reply:
this was added only in may for corporate subs, apparently just to get the tests working https://github.com/guardian/support-frontend/pull/2484/files#diff-6bc499707cdd7d347bfaa7ce1da6dcf4fc67d86af1c0ca30c801f4e3d1b704f1R64
and changed later to ReaderType 
https://github.com/guardian/support-frontend/pull/2635/files#r463085289

Reply:
If it is always sent, which would make sense, then I think we should remove it